calibration/PARADK/CGINS-PARADK-20542__20220131.csv

```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20542,CC_dark_offset,1.01,source_file PARAD-K_QSP2200_SN_20542_Calibration_2022-01-31.pdf
+20542,CC_scale_wet,9.39E-18,
```

Comment:
good, with CC_dark_offset Probe Dark and CC_scale_wet units Volts/(quanta/cm^2sec)